app/views/inthenewspoets/items/_search_res_item.html.erb

```diff
@@ -1,4 +1,8 @@
 <%= search_result_item_field(locals[:item], "keywords", "Education") %>
 <%# note: below field needs to be parsed further %>
-<%= search_result_item_field(locals[:item], "subjects", "News Items") %>
-
+<div class="search_result_field">
+  <% if locals[:item]["subjects"] %>
+    <% count = locals[:item]["subjects"].count %>
+    News Items: <span><%= count %></span>
+  <% end %>
+</div>
```

Comment:
Probably want to wrap the `if` around the `<div>` too, otherwise it'll be an empty `<div>`